10_datastructures_lab/02.py

```diff
@@ -1,5 +1,27 @@
-"""
-Write a program that prints only unique values
-from sys.argv (without program name)
-"""
-
+'''
+Receives file with words separated by lines.
+Prints anagrams in same line
+Usage: 02.py <file_path>
+'''
+
+import sys
+import os
+if len(sys.argv) != 2 or not os.path.exists(sys.argv[1]):
+    print "Usage: %s <file path>" % sys.argv[0]
+
+dictionary = {}

```

Comment:
A defaultdict would save you the if later on. Define with `dictionary = collections.defaultdict(list)` and then fetching a non existent value yields a new empty list so it's always safe to `dictionary[key].append(word)`